test/unit/math/prim/meta/contains_std_vector_test.cpp

```diff
@@ -2,15 +2,16 @@
 #include <gtest/gtest.h>
 #include <vector>
 
-TEST(MetaTraitsPrimArr, contains_vector_true) {
+TEST(MathMetaPrim, contains_vector_true) {
   using stan::contains_std_vector;
   using std::vector;
 
   EXPECT_TRUE(contains_std_vector<std::vector<double> >::value);
   EXPECT_TRUE(contains_std_vector<std::vector<int> >::value);
   EXPECT_TRUE(contains_std_vector<const std::vector<double> >::value);
   EXPECT_TRUE(contains_std_vector<const std::vector<int> >::value);
-
+  EXPECT_TRUE(contains_std_vector<std::vector<Eigen::VectorXd> >::value);
```

Comment:
@rok-cesnovar, thanks. Just adding a comment letting you know that I see these changes and approve them.